ui/client/containers/AdminPage.js

```diff
@@ -4,17 +4,18 @@ import {withRouter} from "react-router-dom"
 import {Tab, TabList, TabPanel, Tabs} from "react-tabs"
 import "react-tabs/style/react-tabs.css"
 import {nkPath} from "../config"
-import CustomProcesses from "./admin/CustomProcesses"
+import {CustomProcesses, CustomProcessesTabData} from "./admin/CustomProcesses"
 import SearchComponents from "./admin/SearchComponents"
 import Services from "./admin/Services"
 import UnusedComponents from "./admin/UnusedComponents"
+import {Page} from "./Page"
 
 export class AdminPage extends React.Component {
   tabs = [
```

Comment:
Why all components don't have `TabData`? You have changed it only at CustomProcesses, I wonder why?

Reply:
i don't like pattern from "UnusedComponents", makes problems with ts. I changed only what I had to